### PR TITLE
Update homepage + docs 

### PR DESCRIPTION
**Pull request by sachayves**
Update homepage, tightly couple documentation to book, remove outdated information, simplify

### CODE REVIEW

**Review by kdeme — APPROVED** [review]
It is also still confusing that there is the (not up to date) /docs page (accessible true the top menu). Might put new users on the wrong track.

**Comment by sachayves** [comment]
@kdeme so I'm planning on updating that this week. Still strange to me that we have more than one source of truth for the documentation though. Any thoughts as to what should be done?

**Comment by sachayves** [comment]
One option could be to link directly to the book, expand its scope, and use that as the single source of truth. But that doesn't feel quite right to me.

**Comment by kdeme** [comment]
> One option could be to link directly to the book, expand its scope, and use that as the single source of truth. But that doesn't feel quite right to me.

The quickest would be to just link to the book as it is now.

Perhaps better is that the docs page could be reworked to hold whichever introductory & general information that is still relevant there.
And then add a link to the current book. (And in the future other links to other books or other sources of docs, like of the individual libraries). But, we need to be careful not to lose the focus on nimbus-eth2 information.

**Comment by sachayves** [comment]
@kdeme so i think the documentation page provides a useful introduction to the project as a whole, would be a shame to lose that.

I want to keep the link as is, and push updates to the docs page (tightly couple it with the book). Working on that now.

File: source/docs/building.md [hunk 1]
```diff
@@ -106,4 +106,4 @@ build/state_sim --slots=384 --validators=20000 --attesterRatio=0.66 --json_inter
 
 ### Medalla Ethereum 2.0 Testnet
```
Comment (kdeme) [inline source/docs/building.md:107]: Perhaps this should be on top of this page.

File: source/docs/building.md [hunk 2]
```diff
@@ -3,17 +3,17 @@ id: building
 title: Getting Started with Nimbus
```
Comment (kdeme) [inline source/docs/building.md:3]: This page has mostly redundant information that is then available in the eth1 and eth2 page. How about we just link to those two pages? And put eth2 on top.
Or make a smaller version for the getting started of eth2 and eth1. 

Either that or, we get a very small subset of the most important "tldr" information. (say just build and run )

File: source/docs/libraries.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+- Cryptographic functions: https://github.com/cheatfate/nimcrypto
+
+- Nim Chronicles - detailed logging for Nim: https://github.com/status-im/nim-chronicles
+
+- Nim Chronos - async Nim: https://github.com/status-im/nim-chronos
+
+- Nim Web3 - https://github.com/status-im/nim-web3
+
+- Nim RLP - encoding / decoding library: https://github.com/status-im/nim-rlp
```
Comment (kdeme) [inline source/docs/libraries.md:25]: This is an archived repo, nim rlp is part of nim-eth repository now.

File: source/docs/building.md (hunk 2 above)
Comment (sachayves) [reply source/docs/building.md:3]: this page is no longer displayed. I split it up in two: eth1 and eth2

File: source/docs/building.md (hunk 1 above)
Comment (sachayves) [reply source/docs/building.md:107]: this page is no longer being displayed

File: source/docs/libraries.md (hunk 3 above)
Comment (sachayves) [reply source/docs/libraries.md]: thanks, fixed

File: source/docs/building.md (hunk 1 above)
Comment (kdeme) [reply source/docs/building.md:107]: Oh right, missed that.

**Pull request merged**